src/textual/cli/previews/easing.py

```diff
@@ -5,7 +5,7 @@
 from textual.app import App, ComposeResult
 from textual.cli.previews.borders import TEXT
 from textual.containers import Container, Horizontal, Vertical
-from textual.reactive import Reactive
+from textual.reactive import Reactive, var, init
```

Comment:
`Reactive.init()` is now a global method in `reactive.py`, just like `var()` and `reactive()`.

`Reactive.var()` was basically a duplicate of the global `var()` method in `reactive.py`, so it now gone.